apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/doedshendelse/DoedshendelseJobService.kt

```diff
@@ -219,6 +225,17 @@ class DoedshendelseJobService(
         return false
     }
 
+    private fun sjekkUnder18aar(doedshendelse: DoedshendelseInternal): Boolean {
+        val person =
+            pdlTjenesterKlient.hentPdlModellFlereSaktyper(
+                foedselsnummer = doedshendelse.beroertFnr,
+                rolle = PersonRolle.BARN,
+                saktype = SakType.BARNEPENSJON,
+            )
+        val aar18 = 18
+        return person.toPerson().foedselsaar < aar18
```

Comment:
Foedselsaar vil vel aldri være `< 18`. Fordeselsår er vel f. eks `2015`?

Reply:
Kanskje vi kan gjøre den vi bruker for alder i servicen litt mer generell.

```
private fun Person.under20PaaDato(dato: LocalDate): Boolean {
    // Dersom vi ikke har en fødselsdato antar vi at personen kan ha bursdag på nyttårsaften,
    // for å sikre at vi får med alle som er under 20 år.
    val benyttetFoedselsdato = foedselsdato ?: LocalDate.of(foedselsaar, 12, 31)

    return ChronoUnit.YEARS.between(benyttetFoedselsdato, dato).absoluteValue < 20
}

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/4309/commits/34816c60ef265e107a5ca7c8f90682ee64a795bd